### PR TITLE
DOC: Add MultiIndex.get_locs to generated reference

### LINKED ISSUE #30235: CI: Doc build failing
```
/home/vsts/work/1/s/pandas/core/indexes/multi.py:docstring of pandas.MultiIndex.rst:91: WARNING: autosummary: stub file not found 'pandas.MultiIndex.get_locs'. Check your autosummary_generate setting.
looking for now-outdated files... none found
```

https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=23019

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
This should fix the failing CI, see https://github.com/pandas-dev/pandas/pull/30226

CLoses #30235

### CODE REVIEW

**Comment by TomAugspurger** [comment]
@jorisvandenbossche I pushed some formatting changes to get this passing.

File: pandas/core/indexes/multi.py [hunk 1]
```diff
@@ -2879,37 +2879,37 @@ def convert_indexer(start, stop, step, indexer=indexer, codes=level_codes):
...
+        MultiIndex.get_loc : Get location for a label or a tuple of labels.
+        MultiIndex.slice_locs : Get slice location given start label(s) and
+                                end label(s).
 
         Examples
         --------
         >>> mi = pd.MultiIndex.from_arrays([list('abb'), list('def')])
 
-        >>> mi.get_locs('b')
+        >>> mi.get_locs('b')  # doctest: +SKIP
```
Comment (TomAugspurger) [inline pandas/core/indexes/multi.py]: These were failing on the `dtype=int64` part. That's only present in the repr on 32-bit systems IIRC, so easier to skip.

**Comment by TomAugspurger** [comment]
Just the 32-bit failure being handled elsewhere. Will merge when travis passes.

**Comment by TomAugspurger** [comment]
Things should be good on master once this is merged, thanks to #30237.

**Comment by jorisvandenbossche** [comment]
Thanks for the fixup!

**Pull request merged**